### DISCUSSION TITLE
Sort options for search results

### DISCUSSION THREAD

**Discussion by mavrik83** [dailydotdev/daily#791] [Feature Requests]
[Referenced here](https://github.com/dailydotdev/daily/discussions/294#discussioncomment-4366846) 

Starting a new issue so this doesn’t get lost in the depths of an unrelated issue.

Being able to sort search results by date, relevance, popularity, etc. would go a long way to improve UX.

**Comment by idoshamun** [comment]
Hi @mavrik83,

Thanks a lot for the suggestion, but let's keep the discussion in the referenced issue.
We try to minimize dups.

**Comment by sabdailydev** [comment]
Updated on the referenced issues as well. We now have filters on our search results. Check them out.

<img width="572" height="381" alt="Screenshot 2025-07-14 at 10 09 55" src="https://github.com/user-attachments/assets/8102685f-c125-4b21-8e21-d8ed8282ab79" />
<img width="571" height="351" alt="Screenshot 2025-07-14 at 10 10 03" src="https://github.com/user-attachments/assets/80c06b54-edef-4ff9-9ed8-51a7279f53b6" />
